.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
-  - "0.10"
-  - "0.12"
-  - "4.1"
```

Comment:
Travis also complains about `4.1` having known problems. Testing newer 4.x versions.

Reply:
4.8 works, along with 6.0 and 8.0.
I'll leave it at that, since 4.x is probably a bit old.
Let me know if you want to change it. (We'd need something larger than 4.2)